example/src/main/java/io/netty/example/http/upload/HttpUploadServerHandler.java

```diff
@@ -224,6 +232,24 @@ private void readHttpDataChunkByChunk() {
                     }
                 }
             }
+            // Check partial decoding for a FileUpload
+            InterfaceHttpData data = decoder.currentPartialHttpData();
+            if (data != null) {
+                if (partialContent == null) {
+                    partialContent = (HttpData) data;
+                    if (partialContent instanceof FileUpload) {
+                        System.out.print("Start FileUpload: " + ((FileUpload) partialContent).getFilename() + " ");
```

Comment:
After looking to others, it seems better to make a log as INFO. I will do the change...